src/Jira/Router.ts

```diff
@@ -60,3 +62,20 @@ export default class JiraRouter {
         return router;
     }
 }
+
+
+export class JiraProvisionerRouter {
+    constructor(private readonly config: BridgeConfigJira, private readonly tokenStore: UserTokenStore) { }
+
+    public getRouter() {
+        const router = Router();
+        router.get("/oauth", this.onOAuth.bind(this));
```

Comment:
also need a route to determine if the user is even logged in

Reply:
Done in 32ac765 for JIRA, github coming up!

Reply:
GitHub done too now.